src/app/test/results/page.tsx

```
@@ -0,0 +1,64 @@
+"use client";
+
+import { useState } from "react";
```

ESLint

[error] 'useState' is defined but never used.

(@typescript-eslint/no-unused-vars)